tests/components/conftest.py

```diff
@@ -4,6 +4,14 @@
 
 import pytest
 
+from tests.components.tts.conftest import (  # noqa: F401, pylint: disable=unused-import
```

Comment:
We should never import fixtures. Instead make functions that we can import in fixtures that we define here.

Reply:
I'll fix that in a follow-up PR